util/reggen/interrupt.py

```diff
@@ -20,18 +20,23 @@ class IntrType(JsonEnum):
 
 class Interrupt(Signal):
 
-    def __init__(self, name: str, desc: str, bits: Bits, auto_split: bool, intr_type: IntrType):
+    def __init__(self, name: str, desc: str, bits: Bits,
+                 auto_split: bool, intr_type: IntrType, intr_default: bool):
```

Comment:
Nit: I'd consider changing the `intr_default` name. It's in `interrupt.py` already, so I don't think we need the prefix! But I took a few seconds to realise that this is essentially a reset value (rather than a type of interrupt or something similar). Would something more like `default_val` make sense?

Reply:
Hah, and *that* name is presumably because `type` is a Python keyword :-)

This looks sensible to me, thanks.